src/main/java/com/powsybl/openloadflow/ac/equations/AbstractHvdcAcEmulationFlowEquationTerm.java

```diff
@@ -42,6 +42,10 @@ public abstract class AbstractHvdcAcEmulationFlowEquationTerm extends AbstractEl
 
     protected final double pMaxFromCS2toCS1;
 
+    protected double frozenP = Double.NaN;
```

Comment:
You are storing state in the equation term which is not aligned with initial design where state is maintained outside of equations